### PR TITLE
5620 spotify all tracks by artist

### LINKED ISSUE #5620: [TRIGGER] Spotify
**Describe the event source. What app is this for, and what event does the trigger correspond to?**
 Spotify. Trigger is, When an artist releases a new song or album then.....

**API Doc**
https://developer.spotify.com/documentation/web-api/reference/#/operations/get-an-artists-albums

### PR DESCRIPTION
**Pull request by alysonturing**
resolves #5620

### CODE REVIEW

**Review by GTFalcao — COMMENTED** [review]
Hi @alysonturing , thanks for the contribution!

This looks good to go, I made just one comment about the version number, but since that has nothing to do with functionality I'm moving this forward for QA

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Spotify_5620.pdf](https://github.com/PipedreamHQ/pipedream/files/10938298/Spotify_5620.pdf)

File: components/spotify/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/spotify",
-  "version": "0.3.7",
+  "version": "0.3.8",
```
[the remark this answers is not in the document]
Comment (GTFalcao) [reply components/spotify/package.json:3]: @alysonturing

[the remark this answers is not in the document]
Comment (alysonturing) [reply components/spotify/package.json]: sorry, I thought it was optional, but no problem, just changed it! thanks

**Review by GTFalcao — APPROVED** [review]

**Pull request merged**